gbpservice/nfp/orchestrator/modules/service_orchestrator.py

```diff
@@ -854,23 +854,6 @@ def delete_user_config(self, event):
                            event_data=request_data,
                            is_poll_event=True, original_event=event)
 
-    def _get_network_function_instance_for_multi_service_sharing(self,
-                                                                 port_info):
-        network_function_instances = (
-            self.db_handler.get_network_function_instances(self.db_session,
-                                                           filters={}))
-        provider_port_id = None
-        for port in port_info:
-            if port['port_classification'] == 'provider':
-                provider_port_id = port['id']
-                break
-        for network_function_instance in network_function_instances:
-            if (provider_port_id in network_function_instance['port_info'] and
-                network_function_instance['network_function_device_id']
-                    is not None):
-                return network_function_instance
-        return None
-
```

Comment:
Why we removed FW-VPN sharing from this, Isn't we are going to submit this ( @hemanthravi )  ?